src/Azure.Functions.Cli/Helpers/GlobalCoreToolsSettings.cs

```diff
@@ -19,6 +19,7 @@ public static WorkerRuntime CurrentWorkerRuntime
                 if (_currentWorkerRuntime == WorkerRuntime.None)
                 {
                     ColoredConsole.Error.WriteLine(QuietWarningColor("Can't determine project language from files. Please use one of [--dotnet-isolated, --dotnet, --javascript, --typescript, --java, --python, --powershell, --custom]"));
+                    throw new InvalidOperationException($"Worker runtime cannot be '{WorkerRuntime.None}'. Please set a valid runtime.");
```

Comment:
Should this be a CLI exception as well?

Reply:
Hi @liliankasem , since it's internal logic I have added an InvalidOperationException, In the main logic I have already added a CliException. If you'd prefer I can change it to Cli.

Reply:
@umangsriv where is this exception caught upstream?

Reply:
Hi @liliankasem, the logic I previously mentioned is internal, is actually executes earlier than the main logic, apologies for that, so I have made the necessary changes to ensure it aligns with the main logic flow.